Lib/calendar.py

```diff
@@ -132,13 +132,17 @@ def monthlen(year, month):
 
 
 def prevmonth(year, month):
```

Comment:
I am skeptical about the change, but the code will look better if you handle the valid cases first.

```python
def prevmonth(year, month):
    if month == 1:
        return year-1, 12
    elif 1 < month <= 12:
        return year, month-1
    else:
        raise IllegalMonthError(month)
```

or for efficiency,

```python
def prevmonth(year, month):
    if month > 1:
        if month <= 12:
            return year, month-1
    elif month == 1:
        return year-1, 12
    raise IllegalMonthError(month)
```

Reply:
Yes, you're right. I'll update that to what you suggested, thanks. 

In the [issue](https://bugs.python.org/issue35406) that I opened about this, one said that: 

> nextmonth() is not a public API. You should not use it.

I don't know what is the signature of a function that shows it is a public API or not. How am I going to understand that I shouldn't use this function directly in my code? Can you help me with this?
Also I wonder why are you skeptical about change. Thanks.